src/cards/TitleCard.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * Title Card class.
+ *
+ * Used for creating a Title Card.
+ *
+ * @class
+ */
+class TitleCard {
+  /**
+   * @type {string[]} An array of area ids.
+   * @private
+   */
+  #areaIds;
+
+  /**
+   * @type {titleCardOptions}
+   * @private
+   */
+  #options = {
+    title: undefined,
+    subtitle: undefined,
+    showControls: true,
+    iconOn: "mdi:power-on",
+    iconOff: "mdi:power-off",
+    onService: "none",
+    offService: "none",
+  };
+
+  /**
+   * Class constructor.
+   *
+   * @param {areaEntity[]} areas An array of area entities.
+   * @param {titleCardOptions} options Title Card options.
+   */
+  constructor(areas, options = {}) {
+    this.#areaIds = areas.map(area => area.area_id);
+    this.#options = {
+      ...this.#options,
+      ...options,
+    };
+  }
+
+  /**
+   * Create a Title card.
+   *
+   * @return {Object} A Title card.
+   */
+  createCard() {
+    /** @type {Object[]} */
+    const cards = [
+      {
+        type: "custom:mushroom-title-card",
+        title: this.#options.title,
+        subtitle: this.#options.subtitle,
+      },
+    ];
+
+    if (this.#options.showControls) {
+      cards.push({
+        type: "horizontal-stack",
+        cards: [
+          {
+            type: "custom:mushroom-template-card",
+            icon: this.#options.iconOff,
+            layout: "vertical",
+            icon_color: "red",
+            tap_action: {
+              action: "call-service",
+              service: this.#options.offService,
+              target: {
+                area_id: this.#areaIds,
```

Comment:
Same problem here.
![image](https://github.com/AalianKhan/mushroom-strategy/assets/39784853/bfdf1caa-a818-4815-989a-414bed6bb309)

Reply:
Fixed in next commit.